Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -20,10 +20,11 @@ namespace Editor.CoreAPI
 {
     public class GameLiftFleetManager
     {
+        public readonly string FleetLocation = "custom-location-1";
```

Comment:
Instead of making this a constant, we should instead return the location in the create fleet response